modules/tinymce/src/themes/silver/main/ts/ui/core/color/Options.ts

```diff
@@ -9,12 +9,16 @@ const foregroundId = 'forecolor';
 const backgroundId = 'hilitecolor';
 
 const defaultCols = 5;
+const minColors = Math.pow(defaultCols, 2);
 
 const calcCols = (colors: number): number =>
-  Math.max(defaultCols, Math.ceil(Math.sqrt(colors)));
+  colors > minColors ? Math.ceil(Math.sqrt(colors)) : defaultCols;
```

Comment:
Since we established the problem was elsewhere, can we please revert this code?

`Math.max(x, Math.ceil(Math.sqrt(y)))` should never cause a problem no matter what `x` and `y` are.

Reply:
Reverted, thanks